main.go

```diff
@@ -46,7 +46,8 @@ func main() {
 	viper.AddConfigPath(".")
 	err := viper.ReadInConfig()
 	if err != nil {
-		fmt.Println(err)
+		setupLog.Error(err, "unable to fetch datadog secret")
```

Comment:
This adds we issue setupLog and we expect to exit here to let user know that datadog secret fetching failed.